README.md

```diff
@@ -10,15 +10,9 @@ Example
 Install
 -------
 
-Just create a `composer.json` file and run the `php composer.phar install` command to install it:
-
-```json
-{
-    "require": {
-        "jakub-onderka/php-console-highlighter": "0.*"
-    }
-}
-```
+Just run the following command to install it:
+
+    composer require --dev php-parallel-lint/php-console-highlighter
```

Comment:
Adding a semi-wildcard version to the command, prevents the version being fixed to the latest minor under 1.0.
Composer treats minors under `1.0` as majors, so updating it becomes more involved.

```suggestion
    composer require --dev php-parallel-lint/php-console-highlighter:"0.*"
```